README.md

```diff
@@ -67,15 +67,15 @@ A small example is provided in the [examples](https://github.com/EOSIO/ual-react
 cd examples
 cp default.env .env
 ```
-Now that you have a ``.env`` file, you can set your chain particulars.  Note that this file will not be version-controlled, nor should it be.
+Now that you have a ``.env`` file, so you can set environment variables for your chain particulars.  Note that this file will not be version-controlled, nor should it be.
```

Comment:
so doesn't read well here and isn't necessary
`Now that you have a ``.env`` file, you can set environment variables for your chain.`

Reply:
agreed, will update